common-pg/src/main/scala/com/socrata/pg/soql/SqlFunctions.scala

```diff
@@ -360,6 +362,44 @@ object SqlFunctions extends SqlFunctionsLocation with SqlFunctionsGeometry with
     ParametricSql(Seq(sql), params)
   }
 
+  def windowOverInfo(pSql: ParametricSql,
+                     fn: FunCall,
+                     rep: Map[QualifiedUserColumnId, SqlColumnRep[SoQLType, SoQLValue]],
+                     typeRep: Map[SoQLType, SqlColumnRep[SoQLType, SoQLValue]],
+                     setParams: Seq[SetParam],
+                     ctx: Sqlizer.Context,
+                     escape: Escape): ParametricSql = {
+
+    fn.window match {
+      case Some(windowFunctionInfo) =>
+        val sqlPartitions = windowFunctionInfo.partitions.foldLeft(Tuple2(Seq.empty[String], pSql.setParams)) { (acc, param) =>
+          val ParametricSql(Seq(sql), newSetParams) = Sqlizer.sql(param)(rep, typeRep, acc._2, ctx, escape)
+          (acc._1 :+ sql, newSetParams)
+        }
+
+        val sqlOrderings = windowFunctionInfo.orderings.foldLeft((Seq.empty[String], sqlPartitions._2)) { (acc, ob) =>
+          val ParametricSql(Seq(sql), newSetParams) = Sqlizer.sql(ob)(rep, typeRep, acc._2, ctx, escape)
+          (acc._1 :+ sql, newSetParams)
+        }
+
+        val sqlFrames = windowFunctionInfo.frames.foldLeft((Seq.empty[String], sqlOrderings._2)) { (acc, ob) =>
+          Sqlizer.sql(ob)(rep, typeRep, acc._2, ctx, escape)
```

Comment:
what's going on here?

Reply:
i must be missing something, looking

Reply:
yep, i forgot that i had not finished while i was waiting for soql-reference.  pushed a commit to fix this.